app/views/layouts/_flash.html.erb

```diff
@@ -1,7 +0,0 @@
-<% flash.each do |name, msg| %>
```

Comment:
@ski-climb looks like somebody else went ahead and converted the flash, as it's in the master already, and just forgot to remove the erb